diff-test/core/README_typescript_boilerplate.md

```diff
@@ -0,0 +1,64 @@
+# node-typescript-boilerplate
```

Comment:
Can we get a real readme of the diff testing in here, or at least delete this file?

Reply:
There is one, I can also delete this file.

https://github.com/cosmos/interchain-security/blob/a774948f10c1d26c1afd4dc7ceaa6d6b2f8d6aa0/diff-test/core/README.md